cli/cmd/upgrade.go

```diff
@@ -587,15 +588,25 @@ func (options *upgradeOptions) fetchIdentityValues(k kubernetes.Interface, idctx
 		}
 	}
 
+	clockSkewDuration, err := ptypes.Duration(idctx.GetClockSkewAllowance())
```

Comment:
This was causing the upgrade tests to fail, as protobuf's `duration` is different from `Duration` in golang, we have to manually convert that using this func, if we are using the value directly. I think this wasn't a problem previously because we were again converting it back into protobuf and using it only there.